### DISCUSSION TITLE
using background scripts in add-ons

### DISCUSSION THREAD

**Discussion by pzkt** [ScratchAddons/ScratchAddons#765] [Help] [answered]
I was wondering what the "correct" way of using background scripts for custom add-ons in the ScratchAddons ecosystem was.
I have seen them used in other add-ons but couldn't get them working myself. Help would be appreciated :)

**Comment by TheColaber** [comment]
Look at the Wiki - https://github.com/ScratchAddons/ScratchAddons/wiki/About-persistent-scripts

**Comment by WorldLanguages** [comment, marked as answer]
Persistent scripts. There are a few quirks, specially if you want your persistent script to talk to a popup, but you should be able to figure it out on the wiki. If you need help please reach out to our Discord https://discord.gg/Ak8sCDQ, if that's not an option for you keep using the discussion tab.

**Reply by pzkt** [reply]
oops, don't know how I missed that. Thanks
